tools/jsonschema/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"ship-it/internal/models"
+
+	"github.com/alecthomas/jsonschema"
+)
+
+func writeJSONSchema(name string, v interface{}) error {
+	file, err := os.Create(fmt.Sprintf("api/%s.json", name))
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	encoder := json.NewEncoder(file)
+	encoder.SetIndent("", "  ")
+
+	return encoder.Encode(jsonschema.Reflect(v))
+}
+
+func main() {
+	// NOTE: After creating a new JSON model type, it must be added to this
+	// map in order to auto-generate JSON schema documents.
+	types := map[string]interface{}{
+		"release": models.Release{},
```

Comment:
Seems like it might be possible to automate this using the `go/importer` package to discover the models that are defined.

Reply:
I started looking into it and went down a rabbit hole, something for another day.

Reply:
Took a quick look, that `importer` pkg seems pretty close. If we could enumerate the exported symbols of a package and filter by struct types only, I think that'd get us there.